### PR TITLE
docs: add an EC2 example with raw pulumi code

### PR DESCRIPTION
**Pull request by agamm**
I didn't find an example for raw pulumi code integrated with sst, I hope this helps!

### CODE REVIEW

**Comment by jayair** [comment]
Thanks!

**Comment by agamm** [comment]
I think something got messed up with the indentation and formatting, trying to fix...

https://github.com/sst/ion/pull/1257

File: examples/aws-ec2-pulumi/package.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "aws-ec2-pulumi",
+  "version": "1.0.0",
+  "description": "",
+  "type": "module",
+  "main": "index.js",
+  "scripts": {
+    "deploy": "go run ../../cmd/sst deploy",
+    "remove": "go run ../../cmd/sst remove",
```
Comment (zvictor) [inline examples/aws-ec2-pulumi/package.json:9]: what is this pointing to?

Comment (jayair) [reply examples/aws-ec2-pulumi/package.json:9]: Should probably be removed but this lets you deploy using the local version of sst.

**Pull request merged**